cobalt/black_box_tests/testdata/scroll.html

```diff
@@ -35,18 +34,109 @@
 <body>
   <div class="app" style="overflow:auto;">
     <div id="row" class="row" style="height: 20rem;">
-      <div id="top_one" class="tile">One</div>
+      <div id="tile" class="tile">One</div>
     </div>
   </div>
 </body>
 <script>
+  // Fail if the test is not finished within 15 seconds.
+  var kTimeout = 30 * 1000;
+  var failTimer = setTimeout(fail, kTimeout);
+
+  function fail() {
+    console.log("Failing due to timeout!");
+    assertTrue(false);
+  }
+
+  function phasename(phase) {
```

Comment:
We should be consistent with naming conventions, probably following go/tsjs-style?expand-zippies=true#naming-rules-by-identifier-type and doing lowerCamelCase